org.bitcoincore.bitcoin-qt.json

```diff
@@ -0,0 +1,87 @@
+{
+    "id": "org.bitcoincore.bitcoin-qt",
+    "runtime": "org.freedesktop.Platform",
+    "sdk": "org.freedesktop.Sdk",
+    "runtime-version": "18.08",
+    "command": "bitcoin-qt",
+    "finish-args": [
+	/* Wayland and Xorg support */
+	"--socket=x11",
+	"--socket=wayland",
+	/* Sound */
+	"--socket=pulseaudio",
+	/* Persistent storage */
+	"--persist=.bitcoin",
```

Comment:
We can't make it somehow use XDG dirs here?

Reply:
How so?

Reply:
https://standards.freedesktop.org/basedir-spec/basedir-spec-latest.html Instead of storing the data in $HOME/.bitcoin you should use $XDG_DATA_HOME or $XDG_CONFIG_HOME.

Reply:
I would have preferred seeing this implemented in the source code instead of having a wrapper not a blocker of course